lib/cucumber/formatter/ansicolor.rb

```diff
@@ -29,26 +56,18 @@ module Formatter
     # * <tt>comment</tt>       - defaults to <tt>grey</tt>
     # * <tt>tag</tt>           - defaults to <tt>cyan</tt>
     #
-    # For instance, if your shell has a black background and a green font (like the
-    # "Homebrew" settings for OS X' Terminal.app), you may want to override passed
-    # steps to be white instead of green.
-    #
-    # Although not listed, you can also use <tt>grey</tt>.
-    #
-    # Examples: (On Windows, use SET instead of export.)
-    #
-    #   export CUCUMBER_COLORS="passed=white"
-    #   export CUCUMBER_COLORS="passed=white,bold:passed_param=white,bold,underline"
-    #
     # To see what colours and effects are available, just run this in your shell:
     #
-    #   ruby -e "require 'rubygems'; require 'term/ansicolor'; puts Cucumber::Term::ANSIColor.attributes"
+    #   ruby -e "require 'rubygems'; require 'cucumber/term/ansicolor'; puts Cucumber::Term::ANSIColor.attributes"
     #
     module ANSIColor
       include Cucumber::Term::ANSIColor
 
+      # :stopdoc:
       ALIASES = Hash.new do |h, k|
-        "#{h[Regexp.last_match(1)]},bold" if k.to_s =~ /(.*)_param/
+        next unless k.to_s =~ /(.*)_param/
```

Comment:
We should probably use the match predicate here as it's more optimal 

See: https://github.com/JuanitoFatas/fast-ruby#stringmatch-vs-stringmatch-vs-stringstart_withstringend_with-code-start-code-end

JuanitoFatas repo is considered the holy grail for all ruby optimisations.

NB: This will **force us** to remain as 2.4+ (We are currently 2.5+), and not go back to 2.3+